lang/python/python-lxml/Makefile

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lxml
```

Comment:
Could you rename this to python-lxml to make it more clear that this is a python lib.